contents/handbook/growth/strategy.md

```diff
@@ -81,6 +81,8 @@ As we grow, it'll get important to work out which teams in PostHog own different
 
 </span>
 
+Following the 1-2-3 framework, we are currently focused on building our team in the first column - _Free_ - and already have Developer Experience and Business Ops people in place. Only after we have brought in someone to take care of Marketing/Developer Evangelism will we then look at recruiting people into the roles in the second and third columns, _Team_ and _Enterprise_. 
```

Comment:
I'd say it's free + team at the moment from our current hiring, as we're in a position to do both well. But enterprise will be further out once those two columns are done.

Reply:
Done!